<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Forensics Data Extractor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tab_results">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_fs">
       <attribute name="title">
        <string>Sélection du dossier</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Sélectionnez le dossier personnel de l'utilisateur à scanner</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="4">
         <widget class="QTextEdit" name="scan_status_text">
          <property name="acceptRichText">
           <bool>false</bool>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>136</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QLineEdit" name="directory_line"/>
        </item>
        <item row="0" column="3">
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>137</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="scan_button">
            <property name="text">
             <string>Scanner</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="browse_button">
            <property name="text">
             <string>Parcourir</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="1" colspan="3">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="cookies_label">
            <property name="text">
             <string>Cookies</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="downloads_label">
            <property name="text">
             <string>Téléchargements</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="forms_label">
            <property name="text">
             <string>Formulaire</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="places_label">
            <property name="text">
             <string>Places</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="search_label">
            <property name="text">
             <string>Recherche</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="signons_label">
            <property name="text">
             <string>Authentifiants</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="downloads_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="search_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="cookies_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="forms_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="places_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="signons_number">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>600</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>Résultats</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="cookies_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>102</width>
             <height>102</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Cookies</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QTableView" name="cookies_view"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="downloads_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>102</width>
             <height>102</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Téléchargements</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QTableView" name="downloads_view"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="forms_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>102</width>
             <height>102</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Formulaires</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTableView" name="forms_view"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="places_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>766</width>
             <height>415</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Surf</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableView" name="places_view"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="search_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>102</width>
             <height>102</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Recherches</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QTableView" name="search_view"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="signons_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>102</width>
             <height>102</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Authentifiants</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableView" name="signons_view"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
